src/clj_momo/lib/es/document.clj

```diff
@@ -256,19 +256,16 @@
   [sort_by sort_order]
   (let [sort-fields
         (map (fn [field]
-               (let [sp (clojure.string/split field #":")]
-                 {(first sp)
-                  {:order (or (second sp)
-                              sort_order)}}))
+               (let [[field-name field-order] (clojure.string/split field #":")]
+                 {(keyword field-name)
```

Comment:
are we forced to use keyword there? I was already bitten by non keyword compatible fields (that could contain `/` for example)